src/middlewared/middlewared/alembic/versions/12.0/2020-05-28_12-16_remove-ds-sysctls.py

```diff
@@ -0,0 +1,41 @@
+"""Remove legacy AD sysctls
+
+Revision ID: 85346ccd33c0
+Revises: a3ac49efb063
+Create Date: 2020-05-28 16:29:33.277126+00:00
+
+"""
+from alembic import op
+import subprocess as subp
+from middlewared.utils import osc
+
+# revision identifiers, used by Alembic.
+revision = '85346ccd33c0'
+down_revision = 'a3ac49efb063'
+branch_labels = None
+depends_on = None
+
+
+TABLE = 'system_tunable'
+
+
+def remove_ad_sysctls():
+    conn = op.get_bind()
+    conn.execute(
+        f'DELETE FROM {TABLE}'
+        ' WHERE tun_var'
+        ' LIKE "freenas.directoryservice%"'
```

Comment:
Are there any other sysctls that we had under freenas namespace that we should/could remove?